src/natcap/invest/ndr/ndr.py

```diff
@@ -47,55 +50,49 @@
                 "units": u.none
             }},
             "about": (
-                "Map representing the spatial variability in runoff "
-                "potential, i.e. the capacity to transport nutrients "
-                "downstream. This can be a quickflow index (e.g. from the "
-                "InVEST Seasonal Water Yield model) or simply annual "
-                "precipitation. The model will normalize this raster by "
-                "dividing by its average value to compute the runoff "
-                "potential index (RPI)."),
-            "name": "Nutrient Runoff Proxy"
+                "Map of runoff potential, the capacity to transport "
+                "nutrients downstream. This can be a quickflow index "
+                "or annual precipitation. Any units are allowed since "
+                "the values will be normalized by their average."),
+            "name": "nutrient runoff proxy"
         },
         "watersheds_path": {
             "type": "vector",
             "projected": True,
-            "fields": {"ws_id": {"type": "integer"}},
             "geometries": spec_utils.POLYGONS,
+            "fields": {},
```

Comment:
is `ws_id` no longer a required field?

Reply:
Correct, there is no use of `ws_id` in `ndr/ndr.py` or `ndr/ndr_core.pyx`.